### ISSUE TITLE
get_results() method not accepting polygons

### ISSUE THREAD

**Issue by RainfallNZ** [tethys-ts/tethysts#88]
get_results() only allows Point geometries to be passed on to utils.spatial_query().

```console
>>> PrecipData = ts.get_results(dataset_id, geometry=geometry, squeeze_dims=True)
ValueError: A station_id, point geometry or a combination of lat and lon must be passed.
```

It looks like that within get_results() 'Polygon' is missing from the allowed geom_type:

https://github.com/tethys-ts/tethysts/blob/8871823f6838c9ced8bd23b05242d0c28d871fd8/tethysts/main.py#L507

The utils.spatial_query() works fine with a polygon input, so get_results() can be side-stepped as a work-around.

**Comment by mullenkamp** [comment]
Thanks Tim! I'll look into and update the package accordingly.

**Comment by mullenkamp** [comment]
And yes, you can first use the get_stations method to query the stations/geometry then pass those stations to the get_results method. 
I was originally going to completely cut out the geometry querying from the get_results method, but I realised there are decent use cases for skipping the get_stations method (e.g. gridded results).

**Comment by mullenkamp** [comment]
Solved with #90
